README.md

```diff
@@ -23,32 +23,21 @@ This pulls submodule dependencies from github.
 Introduction
 ==
 
-### What's New in 6.2
+### What's New in 7.0
 
-**Library Upgrades**
-- We've updated React Native to version 0.55.4.
+**Android Platform Enhancements**
+Mobile SDK now supports
+- AndroidX and Jetpack. See https://developer.android.com/jetpack/.
+- Firebase Cloud Messaging (FCM) push notifications. Because FCM currently works with the GCM infrastructure, no migration is
+necessary at this time.
 
-**Tool Version Upgrades**
-- We now require Cordova CLI 8.0.0.
-- We now support Android Studio 3.1 and Gradle 4.4.
-
-**SmartSync Data Framework Enhancements**
-- The SmartSync Data Framework now saves the error returned when records fail to sync up.
-- A new utility, `MetadataSyncManager`, harnesses the power of SmartSync Data Framework to query Salesforce object metadata.
-- Another new utility, `LayoutSyncManager`, harnesses the power of SmartSync Data Framework to query Salesforce object layouts.
-
-**Other Technical Improvements**
-- We've replaced `IntentService` with `JobService`. This change helps optimize the execution of background tasks, especially on newer Android versions.
-- We’ve given the Switch User screen a face lift.
-- We've consolidated our templates under a [single repository](https://github.com/forcedotcom/SalesforceMobileSDK-Templates).
-- Improvements to sample apps.
-- Various bug fixes.
+**Version Updates**
+- Android target SDK: Android Pie (API 28)
+- Gradle: 4.6
+- Android Studio: 3.2
```

Comment:
@wmathurin  Should I have put cordova version updates here?

Reply:
I guess so (and the RN library change also) looking at https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/v6.0.0/README.md